geomesa-tools/bin/geomesa

```diff
@@ -36,14 +36,36 @@ if [ -z "$GEOMESA_HOME" ]; then
     export PATH=${GEOMESA_HOME}/bin:$PATH
     echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
 else
-    echo >&2 "Using GEOMESA_HOME $GEOMESA_HOME"
+    echo >&2 "Using GEOMESA_HOME as set: $GEOMESA_HOME"
+    read -p "Is this intentional? Y\n " -n 1 -r
```

Comment:
so this will make the user confirm their geomesa home every time? I don't like this...